### ISSUE TITLE
Audio Balance

### ISSUE THREAD

**Issue by gestanks** [AndreMiras/pycaw#81]
Hello, is there a command where I can set the left and right volume settings differently? I need this in my Pyqt5 application, using leftvolume and rightvolume instead of Mastervolume.

**Comment by TJ-59** [comment]
It has been added from #87, but I think it is "per session", not "per device".
You could probably still use it that way with a bit of gymnastics for each session, including the newly created sessions caught with `AudioSessionNotification`'s `on_session_created`.

**Comment by gexgd0419** [comment]
If you want to set the audio balance for an audio device, `IAudioEndpointVolume` has these methods: [`GetChannelVolumeLevel`](https://learn.microsoft.com/en-us/windows/win32/api/endpointvolume/nf-endpointvolume-iaudioendpointvolume-getchannelvolumelevel), [`GetChannelVolumeLevelScalar`](https://learn.microsoft.com/en-us/windows/win32/api/endpointvolume/nf-endpointvolume-iaudioendpointvolume-getchannelvolumelevelscalar), [`SetChannelVolumeLevel`](https://learn.microsoft.com/en-us/windows/win32/api/endpointvolume/nf-endpointvolume-iaudioendpointvolume-setchannelvolumelevel), [`SetChannelVolumeLevelScalar`](https://learn.microsoft.com/en-us/windows/win32/api/endpointvolume/nf-endpointvolume-iaudioendpointvolume-setchannelvolumelevelscalar).

**Comment by AndreMiras** [comment]
Yes indeed this is available per session through `session.channelAudioVolume()` and per device through the channel methods on `device.EndpointVolume`. There's also an example in `examples/channel_audio_volume_example.py`. Since both options are covered in the replies above, I'm closing this as answered.
